src/casa.wtest

```diff
@@ -0,0 +1,36 @@
+import casa.*
+
+test "romper algo de valor 1000" {
+	casaDePepeYJulian.romperEnCosto(1000)
+	assert.equals(true,casaDePepeYJulian.necesitaReparacion())
```

Comment:
Acá aunque esta bien  lo que queres verificar , no es un uso correcto del assert.equals() porque son mensajes booleanos , devuelven Si o No estos mensajes . Por lo cual en estos casos lo adecuado es usar un assert.that (objeto.mensaje()).